packages/zeebe-element-templates-json-schema/src/defs/properties.json

```diff
@@ -345,6 +345,34 @@
             }
           }
         ]
+      },
+      {
+        "if": {
+          "properties": {
+            "binding": {
+              "properties": {
+                "type": {
+                  "const": "zeebe:userTask"
+                }
+              },
+              "required": [
+                "type"
+              ]
+            }
+          },
+          "required": [
+            "binding"
+          ]
+        },
+        "then": {
+          "properties": {
+            "type": {
+              "enum": [
+                "Hidden"
+              ]
+            }
+          }
+        }
```

Comment:
This means that `type` is still optional, but when it is present, it has to take value of `Hidden`. I'd rather expect this to be enforced to avoid ambiguities. Otherwise, the implementation needs to take care of the default type value for `zeebe:UserTask`.

```json
{
  "$schema": "./schema.json",
  "name": "test",
  "id": "test",
  "appliesTo": [
    "bpmn:UserTask"
  ],
  "elementType": {
    "value": "bpmn:UserTask"
  },
  "properties": [
    {
      "binding": {
        "type": "zeebe:userTask"
      }
    }
  ]
}
```